src/app/work-item/work-item-quick-add/work-item-quick-add.component.html

```diff
@@ -32,4 +32,40 @@
       <span (click)="toggleQuickAdd()" class="pull-right closeQuickAdd"><i class="pficon pficon-close"></i></span>
     </div>
   </div>
+</div>
+
+<!-- Quick ADD Template for Board View -->
+<div class="quickAdd_container quickAdd_container2" id="workItemQuickAdd_container" *ngIf="wilistview === 'wi-card-view'" >
```

Comment:
Make separate classes for those two in this case. Only **quick-add_container2** for this one, do not use **quickAdd_container**.